operations/configuration_management/Target_Ansible_Tasks_for_Image_Customization.md

```diff
@@ -3,24 +3,40 @@
 The Configuration Framework Service \(CFS\) enables Ansible playbooks to run against both running nodes \(node personalization\) and images prior to boot\(image customization\).
 See [Configuration Management Use Cases](Configuration_Management.md#use-cases) for more information about image customization and when it should be used.
```

Comment:
In Configuration_Management.md this line should be
OLD: 'This use case enables partitioning a full configuration of a target node.'
NEW: 'This use case enables provisioning a full configuration of a target node.'
I just don't think partitioning is the verb you want.